upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -269,9 +270,42 @@ func buildCloudInstanceGroup(ig *api.InstanceGroup, g *autoscaling.Group, nodeMa
 	return n
 }
 
-func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval time.Duration, k8sClient *k8s_clientset.Clientset) error {
+func (n *CloudInstanceGroup) RollingUpdate(cloud fi.Cloud, force bool, interval time.Duration, headroom int64, k8sClient *k8s_clientset.Clientset) error {
 	c := cloud.(awsup.AWSCloud)
 
+	if headroom > 0 {
```

Comment:
Should we log that we are adding servers, also see my comment about adding a server per az ...  Let's talk about the design.